editor/editor_run.cpp

```diff
@@ -271,8 +265,22 @@ Error EditorRun::run(const String &p_scene, const String &p_write_movie) {
 
 	int instances = EditorSettings::get_singleton()->get_project_metadata("debug_options", "run_debug_instances", 1);
 	for (int i = 0; i < instances; i++) {
+		// Specify the position for current instance.
+		List<String> instance_args = args;
+		if (instance_positions.size() > 0) {
+			Vector2 pos = Vector2();
+			if (i < instance_positions.size()) {
+				pos = instance_positions[i];
+			} else {
+				pos = instance_positions[instance_positions.size() - 1];
+			}
+			pos += screen_rect.position;
```

Comment:
That's gonna add screen position twice when using "top left", "centered" or "maximized"/"fullscreen" and window will end up on the other screen (if it's not opening on the main screen with 0,0 coordinates). So `+= screen_rect.position;` should be either in the `switch` above or `case`s for the other modes should set it to `Vector2()` instead of `Vector2(screen_rect.position)`.

Reply:
Thanks!

Set all modes to Vector2() instead of Vector2(screen_rect.position)